### PR TITLE
Feature/display output

### PR DESCRIPTION
**Pull request by regular-polygon**
Display paginated table based on data extracted from the uploaded file.

### CODE REVIEW

File: react-paw-mailmerge/src/components/ViewOutput.tsx
Comment (Cat-1) [inline react-paw-mailmerge/src/components/ViewOutput.tsx]: Did you want to leave this fake data in here? My impulse would be to remove it.

File: react-paw-mailmerge/src/components/table/TableOutput.tsx
Comment (Cat-1) [inline react-paw-mailmerge/src/components/table/TableOutput.tsx]: What are your thoughts on having the numeric constants towards the top of the page?

**Comment by Cat-1** [comment]
I think there may be adjustments to be made around the CsvResult object (i.e. the object representing the ParsedCsv and its heades) -- you'll want to access the data properties specifically. One thing to be aware of is that if there is more row data than column headers, Papaparse adds a column called __parsed_extra. In your results view (where you show the contents of the data row), you may want to filter that column out.

**Review by Cat-1 — DISMISSED** [review]
Let's get dynamic buttons into master first, then merge master into this branch, fix the merge conflicts (the CsvResult object has changed slightly).

File: react-paw-mailmerge/src/components/ViewOutput.tsx
Comment (regular-polygon) [reply react-paw-mailmerge/src/components/ViewOutput.tsx]: I was on the fence about that. I will remove it because we're ready to populate it with real data.

File: react-paw-mailmerge/src/components/table/TableOutput.tsx
Comment (regular-polygon) [reply react-paw-mailmerge/src/components/table/TableOutput.tsx]: Good catch. It makes sense to move it toward the top.

File: react-paw-mailmerge/src/components/table/TableOutput.tsx
Comment (Cat-1) [inline react-paw-mailmerge/src/components/table/TableOutput.tsx:8]: Is there any functional difference between string[] and Array<string> ? The header in CsvResult is typed as an Array<string>

File: react-paw-mailmerge/src/components/table/TableOutput.tsx
Comment (Cat-1) [inline react-paw-mailmerge/src/components/table/TableOutput.tsx:42]: Just as an FYI, in my branch (it's not committed), I push call the DoMailMerge function here and push down the result. 
`(<TableRow key={index} header={header} rowData={item} maxColumnCount={MaxColumnCount} mailMergeResult={DoMailMerge(item, template)}/>)`

**Review by Cat-1 — APPROVED** [review]
I'd take a look at the feature/mail-merge branch. I don't think you'll have to make any serious adjustments, but take a look first. I'll let you merge when ready.

File: react-paw-mailmerge/src/components/table/TableOutput.tsx
Comment (regular-polygon) [reply react-paw-mailmerge/src/components/table/TableOutput.tsx:8]: `string[]` and `Array<string>` are equivalent.

File: react-paw-mailmerge/src/components/table/TableOutput.tsx
Comment (regular-polygon) [reply react-paw-mailmerge/src/components/table/TableOutput.tsx:42]: It seems like that will perform mail merge eagerly instead of waiting for the user to ask for mail merge output. Have you considered triggering `DoMailMerge` with some kind of user interaction instead?

**Pull request merged**
